pubspec.yaml

```diff
@@ -110,10 +109,10 @@ flutter:
   # "family" key with the font family name, and a "fonts" key with a
   # list giving the asset and other descriptors for the font. For
   # example:
-  # fonts:
-  #   - family: Schyler
-  #     fonts:
-  #       - asset: fonts/Schyler-Regular.ttf
+  fonts:
+    - family: Figtree
+      fonts:
+        - asset: fonts/Figtree-VariableFont_wght.ttf
   #       - asset: fonts/Schyler-Italic.ttf
```

Comment:
Does the Schyler-Italic work when font family is changed to FigTree?

Reply:
I can be done, but there is no Italic in our figma prototype